fhd_core/gridding/visibility_degrid.pro

```diff
@@ -59,11 +59,11 @@ IF n_conj GT 0 THEN BEGIN
     ky_arr[conj_i]=-ky_arr[conj_i]
 ENDIF
 
-xcen=frequency_array#kx_arr
-ycen=frequency_array#ky_arr
+xcen=Float(frequency_array#kx_arr)
+ycen=Float(frequency_array#ky_arr)
  
-x_offset=Floor((xcen-Floor(xcen))*psf_resolution) mod psf_resolution    
-y_offset=Floor((ycen-Floor(ycen))*psf_resolution) mod psf_resolution 
+x_offset=Fix(Floor((xcen-Floor(xcen))*psf_resolution) mod psf_resolution, type=12) ; type=12 is unsigned int
+y_offset=Fix(Floor((ycen-Floor(ycen))*psf_resolution) mod psf_resolution, type=12) ; type=12 is unsigned int
```

Comment:
Aren't all of these changes already in `master`?

Reply:
I think I merged the doubles_frequencies (or whatever it was called) into eor_bubble prior to merging into master, so this is a duplicate of that.

Reply:
Yes, if you *merged* them in, then the changes will be duplicated. That will lead to some problems when you go to merge this into `master`. Since those changes are now in `master`, you could rebase on this branch and remove those commits, then rebase off of `master`.